scripts/breaking_changes_checker/breaking_changes_tracker.py

```diff
@@ -42,63 +42,57 @@ class BreakingChangeType(str, Enum):
 
 class BreakingChangesTracker:
     REMOVED_OR_RENAMED_CLIENT_MSG = \
-        "The client '{}' was deleted or renamed in the current version"
+        "Deleted or renamed client '{}'"
     REMOVED_OR_RENAMED_CLIENT_METHOD_MSG = \
-        "The '{}' client method '{}' was deleted or renamed in the current version"
+        "'{}' client deleted or renamed method '{}'"
     REMOVED_OR_RENAMED_CLASS_MSG = \
-        "The model or publicly exposed class '{}' was deleted or renamed in the current version"
+        "Deleted or renamed '{}' model"
```

Comment:
```suggestion
        "Deleted or renamed model '{}'"
```
We shall keep consistent for the position of `client/method/model` before or after real name.

Reply:
And same for other MSG

Reply:
Made the message more uniform where possible.